pkg/chunk/mem_cache.go

```diff
@@ -146,6 +153,36 @@ func (c *memcache) cleanup() {
 	}
 }
 
+func (c *memcache) cleanupExpire() {
+	var interval = time.Minute
+	if c.cacheExpire < time.Minute {
+		interval = c.cacheExpire
+	}
+	for {
+		var freed int64
+		var cnt, deleted int
+		cutoff := time.Now().Add(-c.cacheExpire)
```

Comment:
put line 164 after Lock?

Reply:
OK, updated. Actually this patch is only a copy-paste of `disk_cache` version.

https://github.com/juicedata/juicefs/blob/2b006074f0db25843ded87cd5e2924031ff10af8/pkg/chunk/disk_cache.go#L333-L335